copr-gpg-download.py

```diff
@@ -1,51 +1,63 @@
 #!/usr/bin/python3
 
+from copr.v3 import Client
+from copr.v3.pagination import next_page
 from copr import create_client2_from_params
```

Comment:
This import is no longer used and can be removed (hint you can run pylint-3 on this file).